apps/library/views/participation.py

```diff
@@ -40,3 +42,16 @@ def perform_create(self, serializer):
                 instance.save()
         except (ValueError, HttpError) as error:
             logger.critical(error, exc_info=True)
+
+        try:
+            send_success = send_application_email(instance)
+            if send_success:
+                instance.sent_to_email = True
+                instance.save()
+        except (
+            AnymailConfigurationError,
+            AnymailError,
+            ValueError,
+        ) as error:
+            msg = f"Не удалось отправить заявку id = {instance.id} на почту.\n{error}"
```

Comment:
```suggestion
            msg = f"Не удалось отправить заявку на участие в фестивали id = {instance.id} на почту {email}."
```
Ошибку передавай следующем параметром в логгер, он сам должен подхватить её. Проверь!